README.md

```diff
@@ -290,36 +290,71 @@ To probe a FortiGate, do something like `curl 'localhost:9710/probe?target=https
 
 The following example Admin Profile describes the permissions that needs to be granted
 to the monitor user in order for all metrics to be available.
```

Comment:
Let's move these two lines to after the table.

Reply:
```suggestion
```